src/logic/safe/store/actions/transactions/fetchTransactions/index.ts

```diff
@@ -14,43 +13,43 @@ import { addOrUpdateCancellationTransactions } from 'src/logic/safe/store/action
 import { addOrUpdateTransactions } from 'src/logic/safe/store/actions/transactions/addOrUpdateTransactions'
 import { AppReduxState } from 'src/store'
 
-const noFunc = () => {}
-
-export default (safeAddress: string): ThunkAction<Promise<void>, AppReduxState, undefined, AnyAction> => async (
+export default (safeAddress: string) => async (
```

Comment:
The return of this function :D

Reply:
it's a curried function, too

![image](https://user-images.githubusercontent.com/3315606/102545066-82e03000-4094-11eb-993c-d7abce2ccf18.png)